src/logging/logging.cpp

```diff
@@ -111,24 +111,12 @@ void InitModules::init_modules(std::initializer_list< const char* > mods_list) {
 }
 
 std::shared_ptr< spdlog::logger >& GetLogger() {
-#if __cplusplus > 201703L
-    [[unlikely]] if (!(logger_thread_ctx.m_logger)) {
-#else
-    if (LOGGING_PREDICT_FALSE(!(logger_thread_ctx.m_logger))) {
-#endif
-        logger_thread_ctx.m_logger = glob_spdlog_logger;
-    }
```

Comment:
Given that it is an open source, is there any urgency to remove this #if flag?

Reply:
Yeah, I don't really understand what's wrong here? Can you post the error you're getting and which profile/compiler you're using? This logic says if it's later than c++17 (right?) than unlikely should exist...isn't that the case? Are we bit passing in the appropriate `-std` flag to the compiler?

Reply:
i just close this PR, since on urgency for now. after investigation, i will reopen this PR , if any wrong found